packages/synthetic-chain/src/lib/core-eval-support.ts

```diff
@@ -147,3 +143,17 @@ export const ensureISTForInstall = async (agd, config, bytes, { log }) => {
   log({ wantMinted });
   await mintIST(addr, sendValue, wantMinted, giveCollateral);
 };
+
+export const readSubmissions = async () => {
+  const files = await fsp.readdir('submission');
+  const names = files.filter(f => f.endsWith('.js')).map(f => f.slice(0, -3));
```

Comment:
Since I'm thinking about maintaining both `.ts` and `.js` files,  will the proposals ever want to be `.ts`?

Reply:
they can't be because they have to run in XS unmodified